cmake/IgnPackaging.cmake

```diff
@@ -228,8 +234,8 @@ endfunction()
 # variable names (like SDFormat_LIBRARIES), the LEGACY_PROJECT_PREFIX argument
 # can be used to specify the prefix of these variables.
 #
-# NOTE: This will be called automatically by ign_create_core_library(~) and
-#       ign_add_component(~), so users of ignition-cmake should not call this
+# NOTE: This will be called automatically by gz_create_core_library(~) and
+#       gz_add_component(~), so users of ignition-cmake should not call this
```

Comment:
this change is out of sequence, but it's not a big deal since it will change soon anyway